imageroot/actions/create-module/30alerts

```diff
@@ -25,6 +25,22 @@ with open('rules.d/nodes.yml', 'w') as f:
       description: "A cluster target has disappeared. An exporter might be crashed.\\n  VALUE = {{ $value }}\\n  LABELS = {{ $labels }}"
 ''')
 
+# Write the loki.yml file
+with open('rules.d/loki.yml', 'w') as f:
+    f.write('''groups:
+- name: Loki
+  rules:
+  - alert: loki_offline
+    expr: up{job="loki"} == 0
+    for: 0m
+    labels:
+      severity: critical
```

Comment:
Maybe severity warning could be enough for it (even if we do not use it for now)